src/requirements/data/index.ts

```diff
@@ -160,6 +162,11 @@ const json: RequirementsJson = {
       schools: ['EN'],
       requirements: envEngineeringRequirements,
     },
+    EP: {
+      name: 'Engineering Physics',
+      schools: ['EN'],
+      requirements: epRequirements,
+    },
```

Comment:
Do you mind renaming to AEP? I believe that's the more common name. The name would be 'Applied and Engineering Physics'. Instead of `epRequirements` for the file name, can we do `aepRequirements`?

Reply:
I believe AEP is the department name, and EP is the undergraduate major.
![image](https://user-images.githubusercontent.com/34158284/115157879-8f33c400-a059-11eb-9d01-020d6ee8f788.png)

Reply:
Ah ok sounds good. I'm ok with leaving it as is then!